lib/orbf/rules_engine/fetch_data/contract_orgunits_resolver.rb

```diff
@@ -49,7 +49,8 @@ def handle_single
       def handle_target_org_units
         target_codes = pyramid.groups(package.target_org_unit_group_ext_ids).map(&:code_downcase)
         target_contracts = @contract_service.for_groups(target_codes, @period)
-        org_units_set = pyramid.org_units_for(target_contracts.map(&:org_unit_id))
+
+        org_units_set = target_contracts.map(&:org_unit)
```

Comment:
Nitpicky, but the map won't return. `Set.new(target_contracts.map(&:org_unit))` or `target_contracts.map(&:org_unit).to_set` would fix it.

Reply:
(Or keep it just an array, because a bit later we make it back into an array)